index.php

```diff
@@ -2,6 +2,8 @@
 // Composer autoloading
 require __DIR__ . '/vendor/autoload.php';
 
+use Symfony\Component\Debug\Debug;
```

Comment:
Depending on comment above: Should only be referenced if debugging is on

Reply:
This is not possible, because `use` statements are only allowed on the top level and not inside scopes like `if` statements.

Reply:
oh, yeah, true that ;)